### ISSUE TITLE
The Serviette Issue

### ISSUE THREAD

**Issue by Bobbanz1** [Skyrat-SS13/Skyrat-TG#13249]
## Round ID: 
Since it's been implemented

## Reproduction:
This line of code right [HERE](https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/serviette.dm#L98)
Currently the code itself is checking if there's no serviettes left in order to allow you to grab one...
To put it into a translated version of the code which people can understand
IF (There are no serviettes left)
Then: Allows the user to grab a serviette from the box (For some reason this makes the object vanish if you var edit it to have none left). 
However IF (There are indeed serviettes left)
Then: Send a chat message telling the user that there are no serviettes left in this box. 

I believe a solution to this would be to simply change 
if (!servleft) 
To 
if (servleft > 0)
Basically it would this way check if the serviettes left is greater than 0.

**Comment by Bobbanz1** [comment]
Another solution would be to change 
`if (!servleft)`
To
`if (servleft != 0)`
